search-endpoint.md

```diff
@@ -444,39 +521,70 @@ The list of returned facet fields will depend on the Discovery configuration: ht
 The JSON response document is as follow
 ```json
 {
+  "id": null,
```

Comment:
is this really needed?

Reply:
I agree, I'll see what we can do about it